README.md

```diff
@@ -1,181 +1,53 @@
-* ARGS - list of arguments given
-
-Note about string arguments and userspace pointers: as pointers are being dereferenced from userspace memory, a malicious program may change the content being read before it actually gets executed in the kernel. Take this into account when doing security related stuff.
-
-Tracee currently supports a subset of system calls events, which can be listed with:
-
-`./start.py -l`
-
-Other supported events are (functions called in kernel space):
 
-* cap_capable - indicates which capabilities were requested
-* do_exit - indicates exited processes
+This will run Tracee with no arguments which will collect all events from all newly created processes and print them as a table to the standard output.
 
+### Understanding the output
 
-Adding new events (especially system calls) to Tracee is straightforward, but one should keep in mind that tracing too many events may cause system performance degradation. Other than that, high event rate can cause samples to be lost (an error message will then be shown as part of the output). For this reason, *read* and *write* syscalls are deliberately excluded from Tracee.
+Each line is a single event collected by Tracee, with the following information:
 
+1. TIME - shows the event time relative to system boot time in seconds
+2. UTS_NAME - uts namespace name. As there is no container id object in the kernel, and docker/k8s will usually set this to the container id, we use this field to distinguish between containers.
+3. MNT_NS - mount namespace inode number.
+4. PID_NS - pid namespace inode number. In order to know if there are different containers in the same pid namespace (e.g. in a k8s pod), it is possible to check this value
+5. UID - real user id (in host user namespace) of the calling process
+6. EVENT - identifies the event (e.g. syscall name)
+7. COMM - name of the calling process
+8. PID - pid of the calling process
+9. TID - tid of the calling thread
+10. PPID - parent pid of the calling process
+11. RET - value returned by the function
+12. ARGS - list of arguments given to the function
 
-## TODO
+### Configuration flags
 
-* Consider tracing commit_creds to detect potential kernel exploits
+Use `--help` to see a full description of all options.
 
-## Known Issues
+## Secure tracing
 
-* Pathname is missing in execve(at) syscalls - Issue #2627 in BCC project
+When Tracee reads information from user programs it is subject to a rare condition where the user program might be able to change the arguments after Tracee has read them. For example, a program invoked `execve("/bin/ls", NULL, 0)`, Tracee picked that up and will report that, then the program changed the first argument from `/bin/ls` to `/bin/bash`, and this is what the kernel will execute. To mitigate this, Tracee also provide "LSM" (Linux Security Module) based events, for example the `bprm_check` event which can reported by tracee and cross-referenced with the reported regular syscall event.
```

Comment:
rare -> race

Reply:
"the program changed" - consider using the words "other thread of the process"

Reply:
This event should never happen unless done by malicious code, so I think race is more apropriate

Reply:
ok